base-theme/layouts/_default/sitemap.xml

```diff
@@ -1,11 +1,13 @@
 {{- $sitemapDomain := getenv "SITEMAP_DOMAIN" | default "ocw.mit.edu" -}}
+{{- $disallowedUrls := slice "categories/" "tags/" "pages/" "resources/" "tags/" -}}
```

Comment:
- `"tags"` twice
- should `"video_galleries"` be here? I noticed `<loc>localhost:3000/video_galleries/</loc>` in my sitemap